lua/lir/actions.lua

```diff
@@ -39,14 +39,16 @@ function actions.edit()
     return
   end
 
+  local keepalt = (vim.w.lir_is_float and '') or 'keepalt '
```

Comment:
Probably don't need the space at the end, but I kept it in anyway.

Reply:
The space at the end was unnecessary.
Can I have it removed?

Reply:
> The space at the end was unnecessary.
> Can I have it removed?

Yes. Done.